app/design/frontend/Magento/luma/Magento_Sales/web/css/source/_module.less

```diff
@@ -343,16 +343,22 @@
         }
 
         .product-item-name {
-            display: inline-block;
+            float:left;
+            width:calc(100% - 20px);
```

Comment:
Please add a space after ":" according to Magento Less coding standard: https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-less.html#properties-colon-indents

Reply:
Ok